main.js

```diff
@@ -0,0 +1,20 @@
+const icon = document.querySelector('.icon-menu');
+const menulist = document.querySelector('.internal-link-mob');
+const menu = document.querySelectorAll('.menu');
+const main = document.querySelector('.main');
+
+const open = () => {
+  menulist.classList.add('internal-link-mobActive');
+  main.style.opacity = '0.2';
+  main.style.backdropfilter = 'blur(8px)';
+};
+
+const close = () => {
+  menulist.classList.remove('internal-link-mobActive');
+  main.style.opacity = '1';
+};
+
+icon.addEventListener('click', open);
+menu.forEach((menuLink) => {
+  menuLink.addEventListener('click', close);
+});
```

Comment:
- [ ] Great job implementing the functionality for the hamburger. But kindly refer to the images below and increase the blur effect in the background so it resembles the Figma template. As you can see the Figma mobile menu has a much darker background with a less visible backdrop.

> Figma Design

![image](https://user-images.githubusercontent.com/90556221/181234251-f2d30c4d-24e7-48e6-b73c-3c6ec74bd0a0.png)

> Your Design

![image](https://user-images.githubusercontent.com/90556221/181234332-40def86a-560b-4125-bcfb-619b0e3c3819.png)

- [ ] Additionally, kindly also implement the black horizontal line at the end of the Mobile menu like I've highlighted in the Figma template above.